src/common/options.cc

```diff
@@ -2198,6 +2198,11 @@ std::vector<Option> get_global_options() {
     .add_service("mon")
     .set_description(""),
 
+    Option("paxos_service_trim_max_multiplier", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .add_service("mon")
+    .set_description(""),
```

Comment:
can set_min(0) and set_flag(Option::FLAG_RUNTIME) to indicate that it can be changed online (since these are read directly from the conf each time and are basic types runtime updates automatically work)